paint-collector-app.php

```diff
@@ -1,23 +1,28 @@
 <?php
 
-$db = new PDO (
-    'mysql:host=DB;dbname=collector_app',
-    'root',
-    'password'
-);
-
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-
-$query = $db->prepare ('SELECT `brand`, `name`, `color`, `base`, `quantity_left`, `purchase_date` FROM `paints`');
-
-$result = $query->execute();
-
-if (!$result) {
-    echo 'You broke it';
-} else {
-    $table = $query->fetchAll();
-    echo '<pre>';
-    var_dump($table);
-    echo '</pre>';
+function returnDb()
+{
+    $db = new PDO (
+        'mysql:host=DB;dbname=collector_app',
+        'root',
+        'password'
+    );
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
 }
 
+function populateTable (array $table) {
+    $row = '';
+    foreach ($table as $key => $value) {
```

Comment:
There's no need to do `$key=>$value` if you're not actually going to use the `$key`.

Reply:
Removed $key=> as unneeded.